### DISCUSSION TITLE
Set image as default (main) to display

### DISCUSSION THREAD

**Discussion by AEK-BKF** [spatie/laravel-medialibrary#2116] [Q&A] [answered]
Hello,
Let's say I have a model x which has many images,so is there any way to set an image as default for display ?
Something like this :
`x->getMedia('images')->default();`

Thanks

**Comment by freekmurze** [comment, marked as answer]
This currently is not supported. I think you should handle this in your own project.

**Reply by AEK-BKF** [reply]
Thank you.
